### PR TITLE
Adds CameraOrthoPrefab

### PR DESCRIPTION
**Pull request by mtsr**
I wanted to be able to set the CameraOrtho world coordinates in a Prefab, so I created a quick CameraOrthoPrefab.

### CODE REVIEW

File: amethyst_utils/src/ortho_camera.rs [hunk 1]
```diff
@@ -86,6 +89,35 @@ impl Component for CameraOrtho {
     type Storage = DenseVecStorage<Self>;
 }
 
+/// CameraOrtho prefab
```
Comment (jojolepro) [inline amethyst_utils/src/ortho_camera.rs]: Would be nice to have a tiny bit more doc.

Comment (mtsr) [reply amethyst_utils/src/ortho_camera.rs]: Heh, I'lll try the derive. Hadn't seen that.

Comment (mtsr) [reply amethyst_utils/src/ortho_camera.rs]: Do you think it still needs docs if it's just a derive?

**Comment by torkleyy** [comment]
@mtsr Unfortunately there's a conflict with the changelog (yes, this happens too often, I'll fix it soon). Would you mind rebasing this PR so we can merge it?

**Comment by mtsr** [comment]
Just rebased it.

**Comment by LucioFranco** [comment]
bors r=azriel91,jojolepro

**Pull request merged**